_posts/2019-06-18-list-strategy.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: posts
+title: "Automatic execution strategy selection in Bazel 0.27"
+authors:
+  - ishikhman
+---
+**tl;dr**: Exciting news! After [0.27](https://blog.bazel.build/2019/06/17/bazel-0.27.0.html) release Bazel will *auto select* a suitable execution strategy if none is provided. Customization and strategy enforcement will still be possible and will be much easier to configure.
```

Comment:
Instead of this tl;dr, I would prefer an introduction sentence or two that really explain what this blog post is about and why people should read it. For example:
 
> When Bazel executes commands that are a part of the build, such as compiler and linker invocations, test runs and so on, it has a choice on how to execute those commands (also called _actions_): locally, remotely, in a sandbox, and so on. This is controlled by [execution strategies](link). Starting with 0.27 release, we implemented a feature in Bazel that will allow it to *auto select* a suitable execution strategy, eliminating the need for manual configuration via command line flags in most cases; and even in cases when customization and strategy enforcement is needed, it is still fully possible and became much simpler.

wdyt?

Reply:
I think it's a great introduction, but I'd still like to keep a short 'tl;dr', so I adjusted it a bit and added the introduction you've suggested.

Reply:
I'd advise to go with an introduction instead of an tl;dr;. When I wrote my first blog post I also wanted to have a tl;dr; but after shipping found that on the Bazel blog overview page it will display the first paragraph and one really does not want this to be a tl;dr;. See https://blog.bazel.build for what I mean.

Reply:
I've seen the blog and reviewed a few previous posts before writing this one, I have also tested it locally to make sure that it looks like I want it to look ;)

I guess, this is a matter of taste. When I am reading blog posts I am truly grateful when an author added a short summary in tl;dr.

Reply:
I am with buchgr@ here. As it is now, introduction and tl;dr tells you exact same things in slightly different words, and introduction is less terse.